lib/wind/utf8.c

```diff
@@ -220,8 +237,8 @@ wind_ucs4utf8(const uint32_t *in, size_t in_len, char *out, size_t *out_len)
             default:
                 break;
 	    }
+	    out += len;
```

Comment:
What we could have done instead is `if (!out) continue;`, then deindent all the code in this `if` block, which would make it more readable, but this will do.